scripts/openyurt_deployer/node/knative.go

```diff
@@ -0,0 +1,158 @@
+package node
+
+import (
+	"github.com/vhive-serverless/vhive/scripts/openyurt_deployer/logs"
+)
+
+// func (node *Node) ParseSubcommandKnative(args []string) {
+// 	nodeRole := args[0]
+// 	operation := args[1]
+
+// 	// Check nodeRole
+// 	if (nodeRole != "master") && (nodeRole != "worker") {
+// 		logs.InfoPrintf("Usage: %s %s <master | worker> init [parameters...]\n", os.Args[0], os.Args[1])
+// 		logs.FatalPrintf("Invalid nodeRole: <nodeRole> -> %s\n", nodeRole)
+// 	}
+
+// 	// Check operation
+// 	if operation != "init" {
+// 		logs.InfoPrintf("Usage: %s %s %s init [parameters...]\n", os.Args[0], os.Args[1], nodeRole)
+// 		logs.FatalPrintf("Invalid operation: <operation> -> %s\n", operation)
+// 	}
+
+// 	// Parse parameters for `knative master/worker init`
+// 	var help bool
+// 	knativeFlagsName := fmt.Sprintf("%s yurt %s %s", os.Args[0], nodeRole, operation)
+// 	knativeFlags := flag.NewFlagSet(knativeFlagsName, flag.ExitOnError)
+// 	knativeFlags.BoolVar(&help, "help", false, "Show help")
+// 	knativeFlags.BoolVar(&help, "h", false, "Show help")
+// 	knativeFlags.StringVar(&node.Configs.Knative.KnativeVersion, "knative-version", node.Configs.Knative.KnativeVersion, "Knative version")
+// 	knativeFlags.StringVar(&node.Configs.Knative.IstioVersion, "istio-version", node.Configs.Knative.IstioVersion, "Istio version")
+// 	knativeFlags.StringVar(&node.Configs.Knative.MetalLBVersion, "metalLB-version", node.Configs.Knative.MetalLBVersion, "MetalLB version")
+// 	knativeFlags.BoolVar(&node.Configs.Knative.VHiveMode, "vhive-mode", node.Configs.Knative.VHiveMode, "vHive mode")
+// 	knativeFlags.Parse(args[2:])
+// 	// Show help
+// 	if help {
+// 		knativeFlags.Usage()
+// 		os.Exit(0)
+// 	}
+
+// 	var vHiveMode string
+// 	if node.Configs.Knative.VHiveMode {
+// 		vHiveMode = "true"
+// 	} else {
+// 		vHiveMode = "false"
+// 	}
+
+// 	node.InstallKnativeServing()
+// 	node.InstallKnativeEventing()
+
+// 	logs.SuccessPrintf("Init Knative Successfully! (vHive mode: %s)\n", vHiveMode)
+// }
+
+// Install Knative Serving
+func (node *Node) InstallKnativeServing() {
```

Comment:
these scripts are duplicates of the ones in vHive. also make sure that you use the same Knative configs (and the same versions!)